src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -61,9 +61,9 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
 
         private async Task RemoveFilesAsync(Manifest newManifest)
         {
-            IEnumerable<string> prevFiles = await GetAllManifestFilesWithVersionsAsync(_manifest).ConfigureAwait(false);
-            IEnumerable<string> newFiles = await GetAllManifestFilesWithVersionsAsync(newManifest).ConfigureAwait(false);
-            IEnumerable<string> filesToRemove = prevFiles.Where(f => !newFiles.Contains(f)).Select(f => f.Substring(0, f.LastIndexOf(Path.DirectorySeparatorChar)));
+            IEnumerable<Tuple<string, string>> prevFiles = await GetAllManifestFilesWithVersionsAsync(_manifest).ConfigureAwait(false);
```

Comment:
Use ValueTuples instead of Tuple.  It's so much more readable.

Reply:
(This goes for every other Tuple in this PR)

Reply:
Ok, never used it.. Let me read about it

Reply:
Not available on .NET Framework 4.6 and netstandard1.5 which are the ones we are targeting. I could add the Nuget package but decided to replace this Tuple<string, string> for a Type instead.